### ISSUE TITLE
VAT Registration Validation through VIES web service does not retrieve validation result

### ISSUE THREAD

**Issue by AndreyHrynyuk** [microsoft/ALAppExtensions#4663]
Hi,

It seems that VAT registration No. validation via VIES web service does not work anymore.
The VIES validation is enabled in setup and the request is sent to web service but the system fails to parse the response (the status remains not verified despite the actual result from VAT VIES http://ec.europa.eu/taxation_customs/vies/services/checkVatService).
By sending the request manually we can see that the service is available and response is correct (valid = false in this case):
![image](https://user-images.githubusercontent.com/36296723/67666585-c64d7b80-f974-11e9-9400-f15113754c03.png)
![image](https://user-images.githubusercontent.com/36296723/67666760-26dcb880-f975-11e9-9db7-d473be4060d8.png)

The debugging shows that the problem might be in the Codeunit 249  "VAT Registration Log Mgt." in the function LogVerification. 
![image](https://user-images.githubusercontent.com/36296723/67666941-820eab00-f975-11e9-8572-049d361341d4.png)

The highlited line cannot retrieve the result from xml with a parameters specified (path = 'descendant::vat:valid' ) and therefore the log is not updated no matter the result (expected - invalid).

Could you please check this issue? 

Thanks.

**Comment by AlexanderYakunin** [comment]
@JesperSchulz

**Comment by maksym-ch** [comment]
Hi guys, any update on it?

**Comment by JesperSchulz** [comment]
Sorry about the delayed reply. A wrong configuration swallowed all my notifications from this repository. I will ask our global team to look into you question asap!

**Comment by JesperSchulz** [comment]
@PredragMaricic, could you look into this issue?

**Comment by JesperSchulz** [comment]
These are busy days, but let me try again :-) @PredragMaricic, could you pick this issue up?

**Comment by sorenfriisalexandersen** [comment]
We have not been able to reproduce this error. Could you please log a support ticket for further investigation?
